nautobot/extras/api/views.py

```diff
@@ -441,9 +441,18 @@ def sync(self, request, pk):
             raise CeleryWorkerNotRunningException()
 
         repository = get_object_or_404(GitRepository, id=pk)
-        repository.sync(user=request.user)
-        return Response({"message": f"Repository {repository} sync job added to queue."})
 
+        job_result = repository.sync(user=request.user)
+
+        return Response({
+                "message": f"Repository {repository} sync job added to queue.",
+                "repository_id": repository.id,
+                "repository_name": repository.name,
```

Comment:
Probably not needed since you have to know the `repository.id` in order to call this endpoint in the first place?

Reply:
True; but when I update e.g. a `Prefix` via the API I get a lot back that I already sent in the request...